src/lib/components/Cytoscape.react.js

```diff
@@ -676,7 +785,32 @@ Cytoscape.propTypes = {
          */
         boundingBox: PropTypes.object,
     }),
-
+    /**
+     * Define a custom context menu
+     */
+    contextMenu: PropTypes.arrayOf(
+        PropTypes.shape({
+            id: PropTypes.string.isRequired,
+            content: PropTypes.string.isRequired,
+            tooltipText: PropTypes.string,
+            coreAsWell: PropTypes.string,
+            selector: PropTypes.string,
+            onClick: PropTypes.string,
+            onClickCustom: PropTypes.string,
```

Comment:
Each of these pieces can get its own docstring as well, which will be carried over to the codegen